src/test/java/cloud/prefab/client/integration/IntegrationTestDescriptor.java

```diff
@@ -0,0 +1,64 @@
+package cloud.prefab.client.integration;
+
+import cloud.prefab.client.PrefabCloudClient;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.junit.jupiter.api.function.Executable;
+
+public class IntegrationTestDescriptor {
+
+  private final String name;
+  private final String client;
+  private final String function;
+  private final Map<String, String> clientOverrides; // unused?
```

Comment:
Unused for now, yes, but there will likely be scenarios as we build up the integration test data where we will need to update settings.